test/visibility/import/edgeCases/anotherMultipleDef.good

```diff
@@ -1 +1,2 @@
-type method A.foo() for class A
+anotherMultipleDef.chpl:1: error: symbol A is multiply defined
+anotherMultipleDef.chpl:7: note: also defined here
```

Comment:
If the public use had not been added, this would start preferring module A's foo?  Is there an obvious reason to update the test by adding the public rather than updating the .good?  (particularly since we have lots of tests added to this PR that check for multiple definitions).  Having the test tell us which case won seems more interesting to me.

And am I correct that the reason the behavior is changing here because `import` is no longer introducing a shadow scope?

Reply:
Ah, nevermind the first paragraph, just saw that a clone of this test was added for the case I was curious about.

Reply:
(Really fiddly, but I'd probably make the source code change to the code clone so that this test's history shows the change in behavior over time without source changes).  That said, I also feel like the clone isn't strictly necessary since I'm not sure it tests anything that all the new tests don't).

Reply:
> And am I correct that the reason the behavior is changing here because import is no longer introducing a shadow scope?

Yes

> (Really fiddly, but I'd probably make the source code change to the code clone so that this test's history shows the change in behavior over time without source changes). 

I have done so